src/appengine/libs/issue_management/issue_filer.py

```diff
@@ -418,8 +418,15 @@ def file_issue(testcase,
     issue.components.add(policy.unreproducible_component)
 
   issue.reporter = user_email
-  issue.save()
 
+  try:
+    issue.save()
+  except:
+    if policy.fallback_component:
+      issue.components.clear()
+      issue.components.add(policy.fallback_component)
```

Comment:
Awesome start! Let's add one more thing here. 

Let's append this string to the issue body as well:

```
'\n\n**NOTE**: This bug was filed into this component due to permission or configuration issues with the specified component(s) <components>'
```

Put this in a global _CONSTANT and format in the components from before we cleared.

Reply:
> Put this in a global _CONSTANT and format in the components from before we cleared.

Sorry I'm not sure I follow. Do you mean to use something in the policy and then put it in issue.body like the following?

```python
    issue.body += '\n\n' + policy.deadline_policy_message
```

Reply:
Yep exactly :)

Reply:
Done.

Reply:
Please modify the tests to account for this.

Also we need to substitute in the components into the message. Let's make the 'fallback_policy_message' support a "%COMPONENTS%" substitution.

i.e.

Given "Blah blah %COMPONENTS" as "fallback_policy_message", we expand that into "Blah blah Component>Blah, Component2>Blah"

Reply:
Done.